Dockerfile

```
@@ -28,11 +28,12 @@ RUN wget https://github.com/jwilder/dockerize/releases/download/$DOCKERIZE_VERSI
     && tar -C /usr/local/bin -xzvf dockerize-linux-amd64-$DOCKERIZE_VERSION.tar.gz \
     && rm dockerize-linux-amd64-$DOCKERIZE_VERSION.tar.gz
 
-COPY package.json yarn.lock /usr/src/app/
+COPY . /usr/src/app
 RUN yarn install --ignore-engines --frozen-lockfile --production --network-concurrency 1 \
     && rm -rf /var/lib/apt/lists/* \
     && rm -rf ~/.node-gyp \
     && rm -rf /tmp/yarn-*
+RUN yarn build
```

When typescript is moved to a dev dependency this needs to be split if you want to continue copying the node_modules dir from the builder.
Also we can get rid of the `rm`s since the builder image will be trashed at the end anyway.

```suggestion
RUN yarn install --ignore-engines --frozen-lockfile --network-concurrency 1

RUN yarn build

RUN rm -rf node_modules \
    && yarn install --production --frozen-lockfile --ignore-optional
```